src/sentry/api/endpoints/project_details.py

```diff
@@ -237,11 +237,20 @@ def validate_symbolSources(self, sources_json):
             )
 
         try:
-            sources = parse_sources(sources_json.strip())
+            sources = parse_sources(sources_json.strip(), filter_appconnect=False)
             sources_json = json.dumps(sources) if sources else ""
         except InvalidSourcesError as e:
             raise serializers.ValidationError(str(e))
 
+        has_multiple_appconnect = features.has(
+            "organizations:app-store-connect-multiple", organization, actor=request.user
+        )
+        appconnect_sources = [s for s in sources if s.get("type") == "appStoreConnect"]
+        if not has_multiple_appconnect and len(appconnect_sources) > 1:
+            raise serializers.ValidationError(
+                "Multiple AppStoreConnect sources feature not enabled"
```

Comment:
Nitpick: This message is similar to, but different from `AppConnectMultipleSourcesError.message` (`"App Store Connect multiple sources feature not enabled"`).